.env.development

```diff
@@ -14,9 +14,14 @@ USER_INFO_COOKIE_NAME='edx-user-info'
 INTEGRATION_WARNING_DISMISSED_COOKIE_NAME='integration-warning-dismissed'
 SEGMENT_KEY=''
 PORT=8734
+SITE_NAME='edX'
 ALGOLIA_APP_ID=''
```

Comment:
Not on this change but I have wondered if we can a .gitignored file like this for local dev override much like the private.py. I always have this file checked out for algolia and don't want to accidentally check it in you know ...

Reply:
We could likely enable that behavior through frontend-build, e.g. attempt to read a `.gitignore`'d `.env.private` file in webpack.dev.config.js after reading the `.env.development` file.

Do you mind filing a Github issue for this in frontend-build? I'll add it to my frontend-build roadmap of nice-to-have fixes :)

Reply:
yeah let me do that it's a good change to have

Reply:
https://github.com/edx/frontend-build/issues/140